gtclang/src/gtclang/Frontend/GTClangASTConsumer.cpp

```diff
@@ -142,55 +142,69 @@ void GTClangASTConsumer::HandleTranslationUnit(clang::ASTContext& ASTContext) {
     return;
 
   // Compile the SIR using Dawn
-  std::list<dawn::PassGroup> PassGroups;
```

Comment:
I made this `PassGroups` because the convention elsewhere in GTClang was to begin variable names with a capital letter. I prefer the dawn approach because it distinguishes between variables and types

Reply:
https://github.com/MeteoSwiss-APN/dawn/blob/80a61cb5c1488bf604175098e19f345292c493a8/gtclang/src/gtclang/Frontend/GTClangASTConsumer.cpp#L118

https://github.com/MeteoSwiss-APN/dawn/blob/80a61cb5c1488bf604175098e19f345292c493a8/gtclang/src/gtclang/Frontend/GTClangASTConsumer.cpp#L131

just a couple examples out of this file.

Reply:
Oh, hmm then it must be inconsistent anyway. Good to make it more like dawn!